Add documentation for page block duplication (#398)
cortezaproject/corteza-docs: src/modules/integrator-guide/pages/compose-configuration/index.adoc

image::compose-configuration/page-builder-example.png[role="data-zoomable"]

> == Page block Duplication

Reviewer: ```suggestion
== Page Block Duplication
```

> === Page block Duplication

Reviewer: ```suggestion
=== Page Block Duplication
```

> Page blocks can be duplicated, whi bnch makes it a breeze to reuse a page block.

Reviewer: ```suggestion
Page blocks can be duplicated, whitch makes it a breeze to reuse a page block.
```

This saves you the hassle of having to recreate a page block every time you need to reuse one.

> ==== Copying a page block

Reviewer: ```suggestion
=== Copying a page block
```

A great use case for copying a page block is when you want to use it on a different page in the same namespace or even a different namespace!

> Suppose you have some content that you would need on a separate page, and you want to skip the process of recreating it, you can simply copy the content page block which you would be using in this case, and paste it on the page you want. 

Reviewer: Doesn't this basically say the same thing?

For copying, all you have to do is navigate to the page that holds the page block you are interested in, click the button highlighted below and the page block will be copied to your clipboard.
In the image below, the title of the pages have been highlighted to indicate that they are different pages.

.Copying a page block

> [annotation,role="data-zoomable"]

Reviewer: Annotations is a bit off; not a big deal but do be more accurate next time.

![image](https://github.com/cortezaproject/corteza-docs/assets/12545711/42f3743f-7976-4137-bd3d-41ba82dc49ed)

![image](https://github.com/cortezaproject/corteza-docs/assets/12545711/ad1f0e8c-bee7-49c9-befa-acde51b60225)

----
{
  "image": {
    "rel": "compose-configuration/page-block-copy.png",
    "w": 1920,
    "h": 1080
  },
  "annotations": [
    {
      "x": 275,
      "y": 10,
      "w": 362,
      "h": 50,
      "kind": "box"
    },
    {
      "x": 113,
      "y": 371,
      "w": 20,
      "h": 20,
      "kind": "box"
    }
  ]
}
----

> You can then navigate to any other page to reuse this page block as many times as you want by pasting it, just as you would any other content on your clipboard.

Reviewer: Perhaps worth including some [keyboard macros](https://docs.antora.org/antora/latest/asciidoc/ui-macros/#keybinding) here

.Pasting a page block
[annotation,role="data-zoomable"]
----
{
  "image": {
    "rel": "compose-configuration/page-block-paste.png",
    "w": 1920,
    "h": 1080
  },
  "annotations": [
    {
      "x": 275,
      "y": 10,
      "w": 378,
      "h": 50,
      "kind": "box"

>     }

Reviewer: The annotation is a bit off

![image](https://github.com/cortezaproject/corteza-docs/assets/12545711/07ace707-88f1-485c-a1b7-4fd4aa37364c)

  ]
}
----

> ==== Cloning a page block

Reviewer: ```suggestion
=== Cloning a page block
```

==== Cloning a page block

Cloning a page block is similar to copying, but instead of copying the page block to your clipboard, it will be cloned on the same page.
For example, if for any reason you would like to have a duplicate of a page block on a page, you can do that in the toolbar of the page block by clicking its clone button.

Below, we have a page with a single block, and we want to clone it.
All we have to do is click the clone button highlighted below. 

.Cloning a page block

> [annotation,role="data-zoomable"]

Reviewer: Annotations are a bit off here; not a big deal but be more accurate next time.

![image](https://github.com/cortezaproject/corteza-docs/assets/12545711/b941ae3e-83ab-4a76-8e95-8f7a47627b00)

![image](https://github.com/cortezaproject/corteza-docs/assets/12545711/1dfbb394-a271-4254-908a-18a04624b83b)

![image](https://github.com/cortezaproject/corteza-docs/assets/12545711/a801676e-e32e-4760-b96b-306d762ea660)
